conans/client/conf/__init__.py

```diff
@@ -17,12 +17,12 @@
 _t_default_settings_yml = Template(textwrap.dedent("""
     # Only for cross building, 'os_build/arch_build' is the system that runs Conan
     os_build: [Windows, WindowsStore, Linux, Macos, FreeBSD, SunOS, AIX]
-    arch_build: [x86, x86_64, ppc32be, ppc32, ppc64le, ppc64, armv5el, armv5hf, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, s390, s390x, sh4le]
+    arch_build: [x86, x86_64, ppc32be, ppc32, ppc64le, ppc64, armv5el, armv5hf, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr, s390, s390x, sh4le, xtensalx6]
```

Comment:
``arch_build`` is the architecture of the build machine. Is this architecture one you would be using to actually compile packages, or is this typically a cross-compile one?

Reply:
> `arch_build` is the architecture of the build machine. Is this architecture one you would be using to actually compile packages, or is this typically a cross-compile one?

@memsharded It is architecture for which I will build package (for ESP32, xtensalx6)

Reply:
Ok, then it can be removed from ``arch_build``, as you will never run Conan itself in this platform.

Reply:
@memsharded Okay, one second ;)

Reply:
@memsharded Done

Reply:
@memsharded Do you have any other thoughts on this pull-request ?

Reply:
@memsharded Could you check ? I have updated pull-request to exclude merge conflicts, but there is some migration test failed ...

Reply:
@czoido Could you check this pull-request ?